samples/csharp_dotnetcore/12.customQABot/README.md

```diff
@@ -0,0 +1,88 @@
+﻿# Custom Question Answering
+
+Bot Framework v4 Custom question answering bot sample.
+
+This bot has been created using [Bot Framework][BF], it shows how to create a bot that uses the [Custom question answering feature in Language Service][LS].
```

Comment:
Suggestion: it may be good if the 'Custom question answering feature in Language Service' link in this paragraph points to docs rather than Language Studio to help users get started (it is OK to leave the link in the next paragraph pointing to Language Studio)